source/btle/btle_discovery.cpp

```diff
@@ -44,7 +47,7 @@ void bleGattcEventHandler(const ble_evt_t *p_ble_evt)
 
                 case BLE_GATT_STATUS_ATTERR_ATTRIBUTE_NOT_FOUND:
                 default:
-                    sdSingleton.terminateCharacteristicDiscovery();
+                    sdSingleton.terminateCharacteristicDiscovery(BLE_ERROR_NONE);
```

Comment:
Discovery of descriptor require the knowledge of the last ATT handle of a characteristic definition. This information wasn't captured before during characteristic discovery, that's why I was forced to change some things in characteristic discovery. 

Knowing that the characteristic discovery end gracefully allow me to set the last handle of the last characteristic discovered to the last handle of its service.